lib/bloc/backup/backup_bloc.dart

```diff
@@ -528,25 +533,32 @@ class BackupBloc {
   Future testAuth(BackupProvider provider, RemoteServerAuthData authData) {
     return _breezLib
         .testBackupAuth(provider.name, json.encode(authData.toJson()))
-        .catchError((error) {
-      log.info('backupBloc.testAuth caught error: $error');
-
-      if (error is PlatformException) {
-        var e = error;
-        // Handle PlatformException(ResultError, "AuthError", Failed to invoke testBackupAuth, null)
-        switch (e.message) {
-          case _signInFailedCode:
-            throw SignInFailedException(provider);
-            break;
-
-          case _notFoundCode:
-            throw RemoteServerNotFoundException();
-            break;
+        .catchError(
+      (error) {
+        log.info('backupBloc.testAuth caught error: $error');
+
+        if (error is PlatformException) {
+          var e = error;
+          switch (e.message) {
```

Comment:
Shouldn't this be e.code?
I'm a bit confused by the usage of message & code overall. _signInFailedCode is message but has code in it's name etc. Please rename the variables with Code or Message suffix, in doing so these issues should resolve.

Reply:
Updated the names, it is still a bit confusing, i added a comment.